.github/ISSUE_TEMPLATE/release_tracking.md

```diff
@@ -0,0 +1,77 @@
+---
+name: 🚋 Release cycle tracking
+about: Create a new release cycle tracking issue for a minor release
+title: Tasks for v<release-tag> release cycle
+labels: ''
+assignees: ''
+
+---
+
+Please see the corresponding section in [release-tasks.md](https://github.com/kubernetes-sigs/cluster-api/blob/main/docs/release/release-tasks.md) for documentation of individual tasks.  
+
+## Tasks
+
+**Notes**:
+* Weeks are only specified to give some orientation.
+* The following is based on the v1.4 release cycle. Modify according to the tracked release cycle.
+
+Week -3 to 1:
+* [ ] [Release Lead] Set a tentative release date for the minor release
+* [ ] [Release Lead] Assemble release team
+
+Week 1:
+* [ ] [Release Lead] Finalize release schedule and team
+* [ ] [Release Lead] Prepare main branch for development of the new release
+* [ ] [Release Lead] Create a new GitHub milestone
+* [ ] [Communications Manager] Add docs to collect release notes for users and migration notes for provider implementers
+* [ ] [Communications Manager] Update supported versions
+
+Week 1 to 4:
+* [ ] [Release Lead] [Track] Remove previously deprecated code
+
+Week 6:
+* [ ] [Release Lead] Cut the v1.3.1 release
```

Comment:
for all the steps where we have docs we could embed the link to the doc. For example this would link to https://github.com/kubernetes-sigs/cluster-api/pull/7499/files#diff-751bae1afb6862debc4d8d6b2d39bba9d2838e720c37c8b2805138c738843025R149

Reply:
I'll track this as a follow-up. I'm fine with manually verifying 2-3 links and ignoring the linkchecker on merge. But those would be just too many :)

Reply:
SGTM

Reply:
Tracked in https://github.com/kubernetes-sigs/cluster-api/issues/7389

Will follow-up ASAP after this PR merges

Reply:
Change of plans. I already added links. They are pointing to a branch on my fork (this PR) for now. Will change them to point to cluster-api in a follow-up PR